app/javascript/styles/rtl.scss

```diff
@@ -112,6 +112,16 @@ body.rtl {
     margin-right: 6px;
   }
 
+  .fa-ul {
+    margin-left: 0;
+    margin-left: 2.14285714em;
```

Comment:
This number is based on in [font-awesome.css](https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/css/font-awesome.css#L46). It is not something that can be changed by myself.